### ISSUE TITLE
Follow thrownn windows between spaces do not work anymore

### ISSUE THREAD

**Issue by ice-blaze** [ianyh/Amethyst#1066]
**Describe the bug**
When I throw a window in another space, my desktop stay on the initial space. I use the shortcut "Throw focused windows to space left/right". I have the option in general checked.

**Applications:**
All applications.

**Versions:**
 - macOS: 10.15.6
 - Amethyst: 0.15.4
 
**Debug Info**
```
Version: 0.15.4 (91)

OS version: Version 10.15.6 (Build 19G2021)

Has permissions: false

Screens:
	(0.0, 0.0, 1792.0, 1120.0) [(0.0, 0.0, 1792.0, 1120.0)]

Manageable applications:
	talagent (com.apple.talagent)
	AppSSOAgent (com.apple.AppSSOAgent)
	Terminal (com.apple.Terminal)
	Google Chrome (com.google.Chrome)
	Microsoft Teams (com.microsoft.teams)
	Safari (com.apple.Safari)
	Messages (com.apple.iChat)
	Notes (com.apple.Notes)
	IPSecuritas (com.lobotomo.IPSecuritas)
	Finder (com.apple.finder)
	Tunnelblick (net.tunnelblick.tunnelblick)
	Endpoint Security for Mac (com.bitdefender.EndpointSecurityforMac)
	Keynote (com.apple.iWork.Keynote)
	Preview (com.apple.Preview)
	Code (com.microsoft.VSCode)
	System Preferences (com.apple.systempreferences)
	Postman (com.postmanlabs.mac)
	Firefox (org.mozilla.firefox)

Configuration:
layouts: (
)
use-canary-build: 0
window-margins: 0
floating-is-blacklist: 1
restore-layouts-on-launch: 0
float-small-windows: 1
floating: (
)
screen-padding-left: 0
mod2: (
    option,
    shift,
    control
)
mod1: (
    option,
    shift
)
window-resize-step: 5
enables-layout-hud: 1
debug-layout-info: 0
window-minimum-height: 0
window-margin-size: 0
mouse-follows-focus: 0
ignore-menu-bar: 0
screen-padding-bottom: 0
screen-padding-right: 0
focus-follows-mouse: 0
enables-layout-hud-on-space-change: 0
new-windows-to-main: 0
screen-padding-top: 0
follow-space-thrown-windows: 1
window-minimum-width: 0
```

**Additional context**
I only use Amethyst for this thrown window feature. All the reste I disabled it.

**Comment by eb-97** [comment]
had the same problem under catalina 10.15.4
in my case the problem was not amethyst related, but to the _auto-swoosh_ function/parameter
 
After "resetting" this parameter in the terminal via

`defaults write com.apple.Dock workspaces-auto-swoosh -bool NO`
`killall Dock`

and 

`defaults write com.apple.Dock workspaces-auto-swoosh -bool YES`
`killall Dock`

I finally got it working again.

**Comment by ice-blaze** [comment]
@eb-97 I don't know if the issue is fixed. Your workaround worked like a charm but well... it's a work around. Maybe we can have some thoughts from a moderator if this issue should be closed.

**Comment by ianyh** [comment]
Should be fixed by #1333
